src/screens/GameOver/index.tsx

```diff
@@ -18,45 +19,66 @@ export const GameOver = () => {
     };
   }, []);
 
+  const [share, setShare] = useState(false);
+
+  const toggleShare = () => {
+    setShare((prev) => !prev);
+  };
```

Comment:
not necessary but u can simply do 
`
const toggleShare = () => setShare((prev) => !prev);
`
decreased 2 line of code

Reply:
Can further be improved with the use of `useBoolean` hook from usehook-ts library.